common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -312,6 +313,7 @@ def _student_view(self, context, banner_text=None):
             'next_url': context.get('next_url'),
             'prev_url': context.get('prev_url'),
             'banner_text': banner_text,
+            'disable_navigation': not is_user_authenticated,
```

Comment:
Sorry, I'm confused -- does this mean that anonymous users can't navigate the sequence at all except via direct links to individual units?

Reply:
That is the intention, yes.